.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-*                                     @garbas @bastien
```

Comment:
I **wish** i had this one

Reply:
Blame your parents for not having an unusual first name!